app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesActivity.java

```diff
@@ -121,34 +111,39 @@ public void checkInAlertBuilder(final int index) {
             + "Ticket: " + thisAttendee.getTicket().getType())
             .setPositiveButton("OK", (dialog, which) -> {
                 Log.d(TAG, "onClick: inside ok");
-                changeCheckStatus(thisAttendee.getId(), index);
+                attendeesPresenter.toggleAttendeeCheckStatus(index, thisAttendee.getId());
```

Comment:
response will be returned in less time. hence very less chance of list change in that small period which is acceptable. 
Any other way to update the list after toggling?